jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/util/CodeWhispererConstants.kt

```diff
@@ -6,7 +6,14 @@ package software.aws.toolkits.jetbrains.services.codewhisperer.util
 import com.intellij.openapi.editor.markup.EffectType
 import com.intellij.openapi.editor.markup.TextAttributes
 import com.intellij.ui.JBColor
+import groovy.lang.Tuple
```

Comment:
avoid Tuple. if you're keeping it can you confirm it works properly in rider or webstorm